src/firewall/core/io/firewalld_conf.py

```diff
@@ -247,6 +249,20 @@ def read(self):
                 )
             self.set("FlushAllOnReload", str(config.FALLBACK_FLUSH_ALL_ON_RELOAD))
 
+        value = self.get("ReloadPolicy")
+        try:
+            parsed = self._parse_reload_policy(value)
+        except ValueError:
+            value2 = config.FALLBACK_RELOAD_POLICY
+            log.warning(
+                "ReloadPolicy '%s' is not valid, using default " "value %s",
+                value if value else "",
+                value2,
+            )
+        else:
+            value2 = self._unparse_reload_policy(parsed)
```

Comment:
This else block is unnecessary. Just put the value2 assignment inside the try.

Reply:
the try/except should only guard the code that we expect to fail. We don't expect that `self._unparse_reload_policy(parsed)` can ever fail. It MUST not fail. If this ever would raise an exception, this would be a bug we want to catch.